### PR TITLE
Rearranged tutorial lists and added content in Architecture

### PR DESCRIPTION
**Pull request by mjcarroll**

### CODE REVIEW

**Review by andermi — APPROVED** [review]
Looks great!

File: docs/docs/theory.md [hunk 1]
```diff
@@ -1 +1,235 @@
...
+| \(\bf{x}_{COG}\)  | Center of Gravity in Link Frame (x,y,z)    | (0.0, 0.0, -1.2) | m         |
+| \(\bf{x}_{COB}\)  | Center of Buoyancy in Link Frame (x,y,z)   | (0.0, 0.0, -1.2) | m         |
+| \(I_{xx}\)        | Roll Mass Moment of Inertia                |  340.0           | kg m\(^2\)|
+| \(I_{yy}\)        | Pitch Mass Moment of Inertia               |  340.0           | kg m\(^2\)|
+| \(I_{zz}\)        | Yaw Mass Moment of Inertia                 |  600.0           | kg m\(^2\)|
+| \(X_{\dot{u}}\)   | Surge Added Mass (\(\mu_{xx}\))            |       720.00  | kg        |
+| \(X_{\dot{q}}\)   | Surge-Pitch Added Mass MOI(\(\mu_{xq}\))   |       900.00  | kg m      |
+| \(Y_{\dot{v}}\)   | Sway Added Mass (\(\mu_{yy}\))             |       720.00  | kg        |
+| \(Y_{\dot{p}}\)   | Sway-Roll Added Mass MOI (\(\mu_{yp}\))    |      -900.00  | kg m      |
+| \(Z_{\dot{w}}\)   | Heave Added Mass: Doors Closed (\(\mu_{yy}\))|    9330.00  | kg        |
```
Comment (andermi) [inline docs/docs/theory.md:114]: should this be `(\(\mu_{zz}\))` instead of `(\(\mu_{yy}\))`?

File: docs/docs/theory.md [hunk 2]
```diff
@@ -1 +1,235 @@
...
+| \(\bf{x}_{COG}\)  | Center of Gravity in Link Frame (x,y,z)    | (0.0, 0.0, -3.7) | m         |
+| \(\bf{x}_{COB}\)  | Center of Buoyancy in Link Frame (x,y,z)   | (0.0, 0.0, -3.0) | m         |
+| \(I_{xx}\)        | Roll Mass Moment of Inertia                |  3220.0          | kg m\(^2\)|
+| \(I_{yy}\)        | Pitch Mass Moment of Inertia               |  3220.0          | kg m\(^2\)|
+| \(I_{zz}\)        | Yaw Mass Moment of Inertia                 |  10.0            | kg m\(^2\)|
+| \(X_{\dot{u}}\)   | Surge Added Mass (\(\mu_{xx}\))            |       310.00  | kg        |
+| \(X_{\dot{q}}\)   | Surge-Pitch Added Mass MOI(\(\mu_{xq}\))   |      1250.00  | kg m      |
+| \(Y_{\dot{v}}\)   | Sway Added Mass (\(\mu_{yy}\))             |       310.00  | kg        |
+| \(Y_{\dot{p}}\)   | Sway-Roll Added Mass MOI (\(\mu_{yp}\))    |     -1250.00  | kg m      |
+| \(Z_{\dot{w}}\)   | Heave Added Mass (\(\mu_{yy}\))            |        10.00  | kg        |
```
Comment (andermi) [inline docs/docs/theory.md:74]: should this be `(\(\mu_{zz}\))` instead of `(\(\mu_{yy}\))`?

File: docs/docs/theory.md [hunk 3]
```diff
@@ -1 +1,244 @@
...
+description below includes alternative notation for added mass, e.g.  \(\\ \mu_{xx}\), in the style of Newman.
+
+
+### Surface Buoy
+|                  | Description                                               |                  |   Units   |
+|:----------------:|:----------------------------------------------------------|:----------------:|:---------:|
+| \(m\)            | Buoy Mass                                                 |  1400            | kg        |
+| \(V\)            | Displacement (undisturbed buoy)                           |  1.0             | m\(^3\)   |
+| \(\bf{x}_{COB}\) | Center of Gravity in Link Frame (x,y,z)                   | (0.0, 0.0, 2.03) | m         |
+| \(\bf{x}_{COG}\) | Center of Buoyancy in Link Frame (x,y,z)                  | (0.0, 0.0, 2.05) | m         |
```
Comment (andermi) [inline docs/docs/theory.md:34]: `\(\bf{x}_{COB}\)` and `\(\bf{x}_{COG}\)` are flipped

**Comment by mabelzhang** [comment]
It would look good to merge this before the project is presented at the next Gazebo Community Meeting.

**Review by andermi — APPROVED** [review]

**Pull request merged**